### DISCUSSION TITLE
trino + nessie configuration

### DISCUSSION THREAD

**Discussion by JunseoChoJJ** [projectnessie/nessie#8801] [Q&A] [answered]
Hello team,

I was trying to configure trino and spark + nessie catalog and postgresql + s3a on k8s to make a lakehouse
my problem is that trino is not working

trino values.yaml
```
service:
  type: NodePort
additionalCatalogs:
  nessie_catalog: |-
    connector.name=iceberg
    iceberg.catalog.type=nessie
    iceberg.nessie-catalog.uri=https://10.163.20.222:32327/api/v2
    iceberg.nessie-catalog.ref=main
    iceberg.nessie-catalog.default-warehouse-dir=s3a://trino-nessie/
    hive.s3.endpoint
    hive.s3.aws-access-key
    hive.s3.aws-secret-key


```

but turns out that trino is not supporting api/v2 yet 
https://github.com/trinodb/trino/pull/22215

My main question is trino use api/v1 and spark use api/v2 
is this possible?

**Comment by JunseoChoJJ** [comment]
it worked ;;; Is there might be cause a problem when you use spark for api/v2 and trino for api/v1?

**Reply by dimas-b** [reply, marked as answer]
Any client can use any API version supported by the Nessie Server. The server will internally reconcile requests and execute them against the same data model.

**Reply by dimas-b** [reply]
Note, however, that API v1 is deprecated. It is recommended to migrate all clients to v2 (whenever possible).

**Reply by JunseoChoJJ** [reply]
@dimas-b thanks for telling me However, trino is not supporting api v2 yet. 
https://github.com/trinodb/trino/pull/22215
It somehow merged but not sure I can use it or no
